source/GameWindow.cpp

```diff
@@ -46,6 +53,40 @@ namespace {
 
 		return false;
 	}
+
+// The following two functions are only available for Valve Steam Deck support.
+#ifdef __linux__
+	//Files::Read does not work because /sys bytes always returns 4096 for file
+	//size and does not match the contents byte size.
```

Comment:
```suggestion
	// Files::Read does not work because /sys bytes always returns 4096 for file
	// size and does not match the contents byte size.
```